runtime/src/in_mem_accounts_index.rs

```diff
@@ -21,6 +21,7 @@ pub struct InMemAccountsIndex<T: IsCached> {
     map: HashMap<Pubkey, AccountMapEntry<T>>,
     storage: Arc<BucketMapHolder<T>>,
     bin: usize,
+    bucket: Arc<CachedBucket<T>>,
```

Comment:
It looks like the bucket is basically `storage[bin]`, yes? (or similar, based on `InMemAccountsIndex::new()`.) I assume adding this bucket param makes life easier?

Reply:
Yes, I want a direct call to it. Probably will be reworked more. Soon, this struct won't use that storage at all.